src/components/common/buttonCta/buttonCta.tsx

```diff
@@ -0,0 +1,18 @@
+import {ButtonHTMLAttributes, ReactNode} from "react";
+import style from "./buttonCta.module.scss";
+interface Props extends ButtonHTMLAttributes<HTMLInputElement> {
+  text: string;
+}
```

Comment:
button 태그에 적용해줄 속성이기 때문에 이렇게 되어야할 것 같네요~
```suggestion
interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
  text: string;
}
```